<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PRTesterWidget</class>
 <widget class="QWidget" name="PRTesterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PRTester</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stkMain">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageTest">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="TestWidget" name="wgtTest" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>235</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnBack">
           <property name="text">
            <string>&lt; Zur√ºck</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSetup">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="SetupWidget" name="wgtSetup" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnStart">
           <property name="text">
            <string>Los &gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SetupWidget</class>
   <extends>QWidget</extends>
   <header>setupwidget.h</header>
   <container>1</container>
   <slots>
    <signal>canStart(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>TestWidget</class>
   <extends>QWidget</extends>
   <header>testwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>wgtSetup</sender>
   <signal>canStart(bool)</signal>
   <receiver>btnStart</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkAnswer()</slot>
  <slot>checkCanStart()</slot>
 </slots>
</ui>
